src/test/java/org/gridsuite/explore/server/ExploreTest.java

```diff
@@ -456,141 +413,141 @@ public void testCaseCreationError() throws Exception {
     }
 
     @Test
-    public void testCreateScriptContingencyList() throws Exception {
+    void testCreateScriptContingencyList() throws Exception {
         mockMvc.perform(post("/v1/explore/script-contingency-lists/{listName}?&parentDirectoryUuid={parentDirectoryUuid}&description={description}}",
                 "contingencyListScriptName", PARENT_DIRECTORY_UUID, null)
                 .header("userId", USER1)
                 .contentType(MediaType.APPLICATION_JSON)
-                .content("Contingency list content")
+                .content("\"Contingency list content\"")
```

Comment:
Why do you need to change that ?

Reply:
The response mock return JSON, but we were returning just plain text...